### ISSUE TITLE
Unable to change language or subs from movie menu using gamepad

### ISSUE THREAD

**Issue by rgabbo** [jellyfin/jellyfin-web#3755]
**Describe The Bug**
Using Gamepad on Chromium 100, with TV layout enabled, I'm unable to interact with drop-down list to select language or subtitles. The field is highlighted but if I press A on the gamepad nothing happens. All other buttons (Play, Trailer, Back, etc) works fine.

**Steps To Reproduce**

1. Enable TV layout in jellyfin web
2. Enable gamepad support
3. Using gamepad, open a movie menu with more than one language and/or subtitles

**Expected Behavior**
I should be able to select audio track or subtitles

**System (please complete the following information):**
 - Platform: Windows 10
 - Browser: Chromium
 - Jellyfin Version: 10.8.1

**Comment by awsms** [comment]
Mh, gamepad doesn't even work for me, even after enabling it in the Controls settings.

**Comment by rgabbo** [comment]
still present

**Comment by rgabbo** [comment]
still present
